components/BroadcastEmbed.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import SpotifyPodcastEmbed from './SpotifyPodcastEmbed';
+import YouTube from './YouTube';
+import { PastBroadcast } from '../types/broadcast';
+
+interface BroadcastEmbedProps {
+  broadcast: PastBroadcast;
+  embedType: 'youtube' | 'spotify';
+  height?: string | number;
```

Comment:
以下のような形で書き直せませんか？
```suggestion
  height?: number;
```
stringを想定しているところがあったら書き直したいです